### ISSUE TITLE
SwiftMailer support only up to PHP V8.1

### ISSUE THREAD

**Issue by ksar-ksar** [Dolibarr/dolibarr#27714]
### Bug

SwiftMailer support only up to PHP V8.1 and the project is stopped : https://github.com/swiftmailer/swiftmailer
I'm a bit afraid by Dolibarr V19 officially compatible with PHP V8.2, and a lot of users using swiftmailer, will create some issues.

There are a few forks that has been made for PHP V8.2 compatibility like : https://github.com/viaaurea/swiftmailer-legacy
Maybe we can test one of them ?

Or try to move to well known PHP mailer library : https://github.com/PHPMailer/PHPMailer

### Environment Version

V19, Develop

**Comment by JonBendtsen** [comment]
I hope changing mailing library would also indirectly fix issues like #25738 and #23682

**Comment by jtraulle** [comment]
@ksar-ksar Indeed, PHPMailer seems to be a good candidate to replace SwiftMailer.

**Comment by eldy** [comment]
A good question is why do you need the unofficial swiftmailer library ? 
What do you miss in the SMTPs handler library provided by default compared to swiftmailer ?
